docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md

```diff
@@ -12,174 +13,150 @@ ms.assetid: 40a67826-e4df-4f59-a651-d9eb0fdc755d
+| .NET Framework 4.7.2   | On Windows 10 April 2018 Update and Windows Server, version 1803: 461808<br/>On all Windows operating systems other than Windows 10 April 2018 Update and Windows Server, version 1803: 461814 |
+| .NET Framework 4.8     | On Windows 10 May 2019 Update and Windows 10 November 2019 Update: 528040<br/>On Windows 10 and Windows Server, version 1909: 528209<br/>On all other Windows operating systems (including other Windows 10 operating systems): 528049 |
+
+### Minimum version
+
+To determine whether a *minimum* version of the .NET Framework is present, use the smallest `Release` REG_DWORD value for that version from the previous table.
+
+For example, if your application runs under .NET Framework 4.8 or a later version, test for a `Release` REG_DWORD value that is *greater than or equal to* 528040.
+
+| .NET Framework version | Minimum value |
+| ---------------------- | ------------- |
+| .NET Framework 4.5     | 378389 |
+| .NET Framework 4.5.1   | 378675 |
+| .NET Framework 4.5.2   | 379893 |
+| .NET Framework 4.6     | 393295 |
+| .NET Framework 4.6.1   | 394254 |
+| .NET Framework 4.6.2   | 394802 |
+| .NET Framework 4.7     | 460798 |
+| .NET Framework 4.7.1   | 461308 |
+| .NET Framework 4.7.2   | 461808 |
+| .NET Framework 4.8     | 528040 |
 
-#### Specific version
+### Use Registry Editor
 
-To determine whether a *specific* version of the .NET Framework is installed on a particular version of the Windows operating system, test whether the **Release** DWORD value is *equal to* the value listed in the table. For example, to determine whether .NET Framework 4.6 is present on a Windows 10 system, test for the a **Release** value that is *equal to* 393295.
+01. From the **Start** menu, choose **Run**, enter *regedit*, and then select **OK**.
 
-#### Minimum version
+    You must have administrative credentials to run regedit.
 
-To determine whether a *minimum* version of the .NET Framework is present, use the smallest **RELEASE** DWORD value for that version from the previous table. (For convenience, the minimum values are also listed in the table that follows.)
+01. In the Registry Editor, open the following subkey: `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full`. If the `Full` subkey isn't present, then you don't have the .NET Framework 4.5 or later installed.
 
-For example, if your application runs under .NET Framework 4.8 or a later version, test for a **RELEASE** DWORD value that is *greater than or equal to* 528040.
+01. Check for a REG_DWORD entry named `Release`. If it exists, then you have .NET Framework 4.5 or later installed. Its value corresponds to a particular version of the .NET Framework. In the following figure, for example, the value of the `Release` entry is 378389, which is the release key for .NET Framework 4.5.
```

Comment:
This is not the version shown in the figure.

Reply:
Oh yes thank you. I updated the screenshot from the Win7 version after rewriting the text. hahaha!